js/apps/system/_admin/aardvark/APP/react/src/views/query/editor/QueryOptionsTab.tsx

```diff
@@ -310,7 +310,7 @@ const OptimizerRules = () => {
       <MultiSelect
         options={ruleOptions}
         value={ruleOptions.filter(option => {
-          return disabledRules.includes(option.value);
+          return disabledRules?.includes(option.value);
```

Comment:
but given the above change, `disabledRules` should always be an array. So why is this `?.` change required?

Reply:
correct